src/lib/auth/account.ts

```diff
@@ -8,12 +9,21 @@ import { ACCOUNT_SETTINGS_DIR } from '$lib/account-settings'
 import { AREAS } from '$routes/gallery/stores'
 import { GALLERY_DIRS } from '$routes/gallery/lib/gallery'
 
+
 export const isUsernameValid = async (username: string): Promise<boolean> => {
-  return webnative.account.isUsernameValid(username)
+  const session = getStore(sessionStore)
+  return session.authStrategy.isUsernameValid(username)
```

Comment:
with my last change, `session.authStrategy` isn't present at this point in the flow, so I'm going to need to add it to the `sessionStore` a little earlier 🙃

Reply:
updated 👍🏼